### DISCUSSION TITLE
파워 플랫폼 개발자 계정 생성 중 로그인 필요 이슈

### DISCUSSION THREAD

**Discussion by sh0816choi** [fusiondevkr/pl900#10] [1 Q&A] [answered]
Power Apps 개발자 요금제 무료로 시작하기를 누르고, M365 E5 구독에서 만든 아이디/패스워드를 입력한후 
등록 진행 중에 도메인 아래와 같은 로그인창이 발생합니다. (우측 하단 로그인버튼을 누르면 로그인창이 다시 발생)
해결방법이 있을까요?

![캡처](https://user-images.githubusercontent.com/87555967/126023987-d0841333-0624-42cd-9df5-78bde7e2c38a.JPG)

**Comment by sesme100** [comment, marked as answer]
안녕하세요! 위의 로그인 세션문제는 브라우저에 남아 있는 쿠키들과 정보들로 인해서 기존에 마이크로소프트 어플리케이션등에 접근했을때의 정보와 얽히게 되어 생기는 문제 입니다.

그러므로 브라우저의 인터넷 브라우저 설정의 히스토리 정보에서 모든 쿠키등과 인터넷 접속 내용 삭제를 해주신 후 M365 개발자 계정에 접속 후 파워앱스 개발자 계정 신청을 해주시기를 바랍니다.

또는 모든 브라우저를 끄고 난 후 크롬브라우저에서 
Incognito 모드 (시크릿모드) 또는 MS Edge 브라우저에서 In Private 모드로 변경 후 사용하시길 적극 권장드립니다

**Reply by justinyoo** [reply]
@sh0816choi 혹시 위 답변으로 해결이 되었다면 우측 상단에 ✅ 모양의 아이콘을 눌러서 답변을 채택해 주시면 됩니다.

**Reply by sh0816choi** [reply]
@sesme100  크롬 쿠키나 인터넷 접속내용을 삭제했어도 동일 현상이 발생하여 실패했고, MS Edge(별도 시크릿이나 In Private 모드는 안함) 에서 시도해서 진행을 성공했습니다. 답변 감사합니다.

**Reply by sesme100** [reply]
다행이네요. 이러한 현상 원인에 대해 다시 한번 확인해 보겠습니다
